service/docs/source/geopm_pio_const_config.7.rst

```diff
@@ -0,0 +1,80 @@
+geopm_pio_const_config(7) -- Signals for ConstConfigIOGroup
+===========================================================
+
+Description
+-----------
+
+ConstConfigIOGroup implements the
+:doc:`geopm::IOGroup(3) <GEOPM_CXX_MAN_IOGroup.3>` interface to provide a
+predefined set of signals by the user with constant values. This is mainly
+useful to provide configuration information needed by some agents that's
+constant in nature, for example, system characterization information.
```

Comment:
Just to clarify, the reason for saying the signals are provided by the user is to explain that this is a user-defined set of signals. Maybe there's a better way of wording that. Maybe it should be mentioned that the administrator is meant to generate the configuration.

The concern I have with the suggestion is that technically the signals are provided to the IOGroup, not the user (unless I'm misunderstanding what you meant).

Reply:
I think we need to be careful how we document this.  This IOGroup allows for signals to be either: user-defined (i.e., the user wrote a config file and set GEOPM_CONST_CONFIG_PATH to point to the file they wrote) or admin defined (i.e., GEOPM_CONST_CONFIG_PATH unset but the default file exists and has entries), or neither, or both user and admin defined.

It should be clear in the docu that:
   a. This could be leveraged by the user, the sysadmin, neither, or both.
   b. How we decide who wins if both files exist, and the env is set properly, or if only one of the files exists, etc.

Reply:
Yes. That needs to be addressed (realized later that the documentation needed to be updated to reflect the latest changes in the code). Just wasn't sure about your original suggestion about a wording change.